src/app/modules/table2/table.component.html

```diff
@@ -98,6 +98,9 @@
 <div aria-atomic="true" aria-live="assertive" class="sr-only">
   <div *ngIf="tableModel.isLoading">The data table is loading.</div>
 </div>
+<div aria-atomic="true" aria-live="assertive" class="sr-only">
+  <div *ngIf="!tableModel.isLoading">The data table has loaded with {{tableModel.rows.length}} results.</div>
```

Comment:
i think you could put this div inside o the other aria-live div, right below the "loading" message